enacts/config-nma.yaml

```diff
@@ -20,13 +20,19 @@ shapes_adm:
       is_checked: False
 
 # Daily ENACTS Zarr conversion
-rr_mrg_nc_path: /Data/data23/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/
-rr_mrg_resolution: .25
-rr_mrg_chunks:
+rr_mrg_zarr_path: /data/aaron/Ethiopia/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/rr_mrg_ALL-staging/ #this I could delete and we could use the new file path defs in the maprooms?
+nc_path: /Data/data23/NMA_Ethiopia_v7/ALL_NEW/ #top level directory path to find input data
+resolution: .0375
+chunks:
     T: 578
     Y: 25
     X: 31
-rr_mrg_zarr_path: /data/aaron/Ethiopia/NMA_Ethiopia_v7/ALL_NEW/Rainfall/daily/rr_mrg_ALL-staging/
+zarr_path: /data/drewr/Ethiopia/NMA_Ethiopia_v7/ALL_NEW/ #top level directory path to put output file
+data_src: 
+   #var name in file name: [input file path, output file path, var name def in file]
+    precip: [Rainfall/daily/, Rainfall/daily/r0375/rr_mrg_ALL/, precip]
+    tmax: [Temperature/daily/tmax/, Temperature/daily/tmax/r0375/rr_mrg_ALL/, temp]
+    tmin: [Temperature/daily/tmin/, Temperature/daily/tmin/r0375/rr_mrg_ALL/, temp]
```

Comment:
Maybe we should we just mirror the structure of the original ENACTS dataset, so that element 0 and element 1 of each config are the same and can be combined? I imagine someone had a reason to add those extra directory levels.

Reply:
I added the r0375 directory level because when the previous script was run it was run at a different resolution, so I thought it would make sense to have the option to create other directories if someone chooses to rerun the script at a different resolution like I did. As for the rr_mrg_ALL directory level, I cannot say. I just kept that because the previous path included it.

Reply:
Oh, I had it backwards. I thought the one with r0375 in it was the original. I agree with having subdirs for different zarr resolutions.

Not a fan of rr_mrg_ALL. I don't even know what the rr means.

Reply:
I restructured / renamed the temp data to address Rémi's comment.